.env

```diff
@@ -0,0 +1,4 @@
+DATABASE_URL_DEV=postgres://andeladeveloper:''@127.0.0.1:5432/documents-dev
+DATABASE_URL_TEST=postgres://andeladeveloper:''@127.0.0.1:5432/documents-test
```

Comment:
Don't push your .env file to git. Always make sure it's the first thing you ignore in your git ignore file